definitions/bat_apply_test.js

```diff
@@ -3,12 +3,13 @@
 const dfeAnalyticsDataform = require("../");
 
 dfeAnalyticsDataform({
+    disabled: true,

```

Comment:
What is this doing?

Reply:
This is a set of Apply test data that I normally use for testing dfe-analytics-dataform. I switched it off with this parameter to make it faster to spot problems in the compiled SQL as well as less expensive to do test pipeline runs.